packages/protocol/scripts/sourcify-publish.ts

```diff
@@ -0,0 +1,111 @@
+/* tslint:disable no-console */
+import FormData from 'form-data'
+import fs from 'fs'
+import fetch from 'node-fetch'
+import path from 'path'
+import Web3 from 'web3'
+
+/*
+ * A script that reads the artifacts from the build/contracts directory and publish using the sourcify api.
```

Comment:
Since this hasn't been fully tested yet, let's add a comment along the lines of

```
NOTE: this script hasn't been tested yet. It should be manually tested and
potentially adjusted before including it in the release process.
```